src/main/java/org/myrobotlab/service/config/ServiceConfig.java

```diff
@@ -1,19 +1,17 @@
 package org.myrobotlab.service.config;
 
-import java.util.ArrayList;
-
 public class ServiceConfig {
 
-  public String name;
+  // public String name; not normalized
```

Comment:
Why are we removing name ?  all services have a name and a type...

Reply:
The name is and will always be in runtime.yml
runtime has a list of names of services to create
that name is tied to a filename to configure it
the name inside the file is not normalized and deceiving